common/src/commonMain/kotlin/org/hisp/dhis/common/App.kt

```diff
@@ -24,10 +24,11 @@ import androidx.compose.ui.graphics.Color
 import org.hisp.dhis.common.screens.BadgesScreen
 import org.hisp.dhis.common.screens.BottomSheetHeaderScreen
 import org.hisp.dhis.common.screens.ButtonBlockScreen
+import org.hisp.dhis.common.screens.ButtonCarouselScreen
 import org.hisp.dhis.common.screens.ButtonScreen
 import org.hisp.dhis.common.screens.CheckboxScreen
 import org.hisp.dhis.common.screens.ChipsScreen
-import org.hisp.dhis.common.screens.Components
+import org.hisp.dhis.common.screens.Components.*
```

Comment:
avoid using wildcards, I think this will fail the checkstyle

Reply:
done!